### ISSUE TITLE
love.window.showMessageBox() inaccessible to VoiceOver on macOS

### ISSUE THREAD

**Issue by TheQuinbox** [love2d/love#1869]
I'm a blind Love2D programmer, using both the Mac and Windows. For my errorhandler, I have the following:
```lua
function love.errorhandler(msg)
	msg = msg .. "\n" .. debug.traceback()
	if love.audio then
		love.audio.stop()
	end
	return function()
		local pressed = love.window.showMessageBox(
			"Error",
			"An error has occured! Please let me know about this! \n"
				.. msg,
			{ "Quit", "Copy" },
			"error"
		)
		if pressed == 1 then
			love.event.quit()
		elseif pressed == 2 then
			love.system.setClipboardText(msg)
			love.window.showMessageBox(
				"Done",
				"Error information copied to clipboard",
				"info"
			)
			love.event.quit()
		end
		return 1
	end
end
```
Whenever an error occurs on Windows, my screen reader (NVDA) is able to read the dialog, click buttons, etc. However, if it happens on the Mac, VoiceOver isn't able to interact with the dialog at all.
This doesn't seem exclusive to errorhandler either, its the showMessageBox function as a hole.

**Comment by devenvexe** [comment]
Hi there. I looked into this as I am ramping up on a11y support across gaming. I think I got good results from VoiceOver if the MessageBox is not attached to the window. Try changing your call to `showMessageBox` to something like the following
```love.window.showMessageBox("Done", "Error", "info", false)```

**Comment by slime73** [comment]
Interesting, I wonder if the main SDL window is stopping some data from being passed through that's necessary for VoiceOver to work.

I should be able to investigate a little bit pretty soon.

**Comment by slime73** [comment]
I reproduced the issue while using SDL directly, so I made a bug report on their repository.

**Comment by slime73** [comment]
A fix was made in SDL's code by @devenvexe (thanks!) - the next release of love will use the latest version of SDL so it'll have the fix.
